src/ert/ensemble_evaluator/monitor.py

```diff
@@ -31,7 +33,7 @@ class Monitor:
     def __init__(self, ee_con_info: "EvaluatorConnectionInfo") -> None:
         self._ee_con_info = ee_con_info
         self._id = str(uuid.uuid1()).split("-", maxsplit=1)[0]
-        self._event_queue: asyncio.Queue[Union[CloudEvent, CloseTrackerEvent]] = (
+        self._event_queue: asyncio.Queue[Union[Event, CloseTrackerEvent]] = (
```

Comment:
hm, maybe here we can use the same `Event._sentinel` variable instead of `CloseTrackerEvent` or?

Reply:
we probably do not want to import from _ert_forward_model_runner in monitor?
or do you mean that we should write it the same way in monitor?

Reply:
The current way is :100: :+1: